### PR TITLE
New C++ framework (poco)

### PR DESCRIPTION
**Pull request by rayrapetyan**

### CODE REVIEW

File: toolset/setup/linux/frameworks/poco.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+fw_get -o poco_$VERSION.tar.gz http://pocoproject.org/releases/poco-$VERSION/poco-$VERSION-all.tar.gz
+fw_untar poco_$VERSION.tar.gz
+
+cp -R poco-$VERSION-all/ $POCO_HOME
+rm -rf poco-$VERSION-all/
+
+cd $POCO_HOME
+./configure --no-tests --no-samples
+make PageCompiler-libexec XML-libexec JSON-libexec
```
Comment (nbrady-techempower) [inline toolset/setup/linux/frameworks/poco.sh]: makes need quiet option

**Comment by nbrady-techempower** [comment]
@rayrapetyan passes travis and local tests, just needs the quiet option

**Comment by nbrady-techempower** [comment]
@rayrapetyan passes travis and local tests but the make in poco.sh needs a quiet option

**Comment by msmith-techempower** [comment]
Looks good; LGTM!

**Pull request merged**